src/connections/KafkaConsumer.ts

```diff
@@ -4,7 +4,12 @@ export class KafkaConsumerClient {
   private consumer: Consumer;
 
   constructor(kafka: Kafka, groupId: string, retry?: RetryOptions) {
-    this.consumer = kafka.consumer({ groupId, retry });
+    this.consumer = kafka.consumer({
+      groupId,
+      retry,
+      readUncommitted: false,
+      allowAutoTopicCreation: false,
```

Comment:
This was by default `true` and causing the client to automatically create a topic if missing and we don't want that feature since we want to manually create topics with custom config before runners can use the topics